addons/player-thumb/userstyle.css

```
@@ -0,0 +1,10 @@
+.sa-project-thumb {
+  position: absolute;
+  top: 1px;
+  bottom: 1px;
+  left: 1px;
+  right: 1px;
+  width: calc(100% - 1px);
+  height: calc(100% - 1px);
```

Subtracting 1 pixel isn't enough - the thumbnail still covers the bottom and right boder.
```suggestion
  width: calc(100% - 2px);
  height: calc(100% - 2px);
```